src/registry.ts

```diff
@@ -56,6 +56,14 @@ export async function getEntry(
         datakey: toHexString(hashDataKey(dataKey)),
       },
       timeout: opts.timeout,
+      // Transform the response to add quotes, since uint64 cannot be accurately
+      // read by JS so the revision needs to be parsed as a string.
+      transformResponse: function (data: string) {
+        // Change the revision value from a JSON integer to a string.
+        data = data.replace(/"revision":([0-9]+)/, '"revision":"$1"');
```

Comment:
This regex should allow for some whitespace between the colon and the number. It doesn't need to preserve it, though.